mlir/test/Dialect/Affine/SuperVectorize/invalid.mlir

```diff
@@ -0,0 +1,9 @@
+// RUN: mlir-opt %s -verify-diagnostics --affine-super-vectorize=virtual-vector-size=0
+
```

Comment:
Maybe rename this file to something like invalid-zero-size.mlir since it will be hard to use it to test anything else.